src/main/java/com/example/videotecha/service/TheaterService.java

```diff
@@ -0,0 +1,12 @@
+package com.example.videotecha.service;
+
+import com.example.videotecha.model.Theater;
+
+import java.util.List;
+
+public interface TheaterService {
+
+    List<Theater> findAll();
+
+    Theater findById(Long id);
```

Comment:
Be consistent with formatting, if you add an empty line at the beginning of the class, add it at the end as well.